app/services/donated_device_requests_exporter.rb

```diff
@@ -40,22 +40,25 @@ def self.headings
   def render(csv, query)
     csv << self.class.headings
     query.find_each do |request|
-      csv << [
-        request.id,
-        request.created_at,
-        request.school.urn,
-        request.school.computacenter_reference,
-        request.school.responsible_body.computacenter_reference,
-        request.user.full_name,
-        request.user.email_address,
-        request.user.telephone,
-        request.device_types.join(','),
-        request.units,
-      ]
+      request.schools.each do |school_id|
```

Comment:
this should be fine if numbers are small but could pose a bottleneck

doing `schools = School.where(id: request.schools)` is marginally better but still means for every request a query is being made